tests/util/mod.rs

```diff
@@ -137,6 +137,20 @@ impl<'a> TestManager<'a> {
 
         Ok(())
     }
+
+    /// Remove a temporary table from the list of temporary tables.
+    pub fn remove_temporary_table(
+        &mut self,
+        table_name: String,
+    ) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
This function always succeeds. We do not need to use `Result`.

Reply:
I think the return value is unnecessary, so I will remove it.